kit/Cargo.toml

```diff
@@ -6,8 +6,12 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-dioxus = { version = "0.2.4", features = ["desktop"] }
-dioxus-heroicons = { git="https://github.com/Satellite-im/dioxus-heroicons" }
+dioxus = { git="https://github.com/DioxusLabs/dioxus", branch = "master" }
```

Comment:
I feel that it would be safer to specify a `rev` here rather than rely on the `master` branch, which constantly changes. same for the `Cargo.toml` file in `ui/`.

Reply:
Yea it is safer and would also make sure everybody who builds are using the same commit than one person still using a older while another may be using a newer. On top of that, it would prevent possible bugs from being merged in in newert commits.